pom.xml

```
@@ -39,6 +43,11 @@
 			<artifactId>mysql-connector-j</artifactId>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mapstruct</groupId>
+			<artifactId>mapstruct</artifactId>
+			<version>1.5.5.Final</version>
```

Add <mapstruct.version> to properties 
```suggestion
			<version>${mapstruct.version}</version>
```

---

not fixed